### ISSUE TITLE
Need help with some commands.

### ISSUE THREAD

**Issue by Hsinyao99** [McJty/InControl#247]
Hello there, I am having some trouble to disable and restrict certain mobs from spawning, here's some of the command I wrote:

1. Disable the natural spawning of enderman in the minecraft's nether

[
  {
    "mob": "minecraft:enderman",
    "dimension": "minecratf:nether",
    "onjoin": true,
    "result": "deny"
  }
]
(I have tried both with and without the onjoin line.)

And 2. Restrict the spawning of the mob strider when there's already 4 striders per player

[
  {
    "dimension": "minecraft:nether",
    "mob": "minecraft:strider",
    "mincount": {
      "amount": 4,
      "mob": "minecraft:strider",
      "perplayer": true
    },
    "result": "deny"
  }
]

Any feedback would be great. Thank you.
And also, the only command that I'm having success so far was to disable the natural spawning of squid in the minecraft river biome using:

[
  {
    "mob": "minecraft:squid",
    "biome": ["minecraft:river", "minecraft:frozen_river"],
    "result": "deny"
  }
]

**Comment by McJty** [comment]
It's actually 'minecraft:the_nether' (i.e. the_nether instead of nether)

**Comment by Hsinyao99** [comment]
okay.. it works! Thanks! So... I'm assuming for the end dimension should put the_end? How about the place where the end city spawn(not where the dragon fight), are they consider as the same dim? And is there any difference between disabling mob spawn from an end biome (where the end city spawn) and the overworld using the "biome" line. Thank you!

Unrelated question: if I restrict the number of certain passive mob spawn will that cause other passive mob which in the same biome to appear more often, upon entering the chunk for the first time? Is that how the command works? Or technically the mob I restricted count as a spawn before the mod removed it?

**Comment by McJty** [comment]
Yes, it's 'the_end'. And yes, those other islands are the same dimension. Not sure what I mean by your other question.

It is possible that restricting one type of mob will increase the occurance of others

**Comment by Hsinyao99** [comment]
Sorry... I just read the wiki, and I assume that there is no way to prevent an interaction between an held item and a mob right? Since you need both the rightclick.json from Fx Control, and the rule files(mob) from In Control?
